### PR TITLE
CLN: Remove unnecessary io function returns

### PR DESCRIPTION
**Pull request by ihsansecer**
These functions return a function either with empty return statement or doesn’t return. It causes docstring validation error with type RT01 (See #26234).

- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -1394,13 +1394,12 @@ def to_gbq(self, destination_table, project_id=None, chunksize=None,
...
         """
         from pandas.io import gbq
-        return gbq.to_gbq(
-            self, destination_table, project_id=project_id,
-            chunksize=chunksize, reauth=reauth, if_exists=if_exists,
-            auth_local_webserver=auth_local_webserver,
-            table_schema=table_schema, location=location,
-            progress_bar=progress_bar, credentials=credentials,
-            verbose=verbose, private_key=private_key)
+        gbq.to_gbq(self, destination_table, project_id=project_id,
```
Comment (WillAyd) [inline pandas/core/frame.py]: Would be good to take this all the way through. `pandas.io.gbq.to_gbq` also has a return statement, but if that actually doesn't return anything should be removed there as well

**Review by WillAyd — CHANGES_REQUESTED** [review]
Minor comment but otherwise lgtm

**Review by WillAyd — APPROVED** [review]
lgtm @jreback

**Comment by jreback** [comment]
thanks @ihsansecer

**Pull request merged**